### PR TITLE
Bugfix: Fix LocalBitcoins orderbook path issue

### PR DESCRIPTION
**Pull request by thrasher-**
# PR Description

Noticed that LocalBitcoins had a pathing issue for orderbook requests when reviewing log output. This PR addresses that.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Travis with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/localbitcoins/localbitcoins_test.go [hunk 1]
```diff
@@ -102,6 +102,19 @@ func TestGetTrades(t *testing.T) {
 	}
 }
 
+func TestGetOrderbook(t *testing.T) {
+	ob, err := l.GetOrderbook("AUD")
```
Comment (MadCozBadd) [inline exchanges/localbitcoins/localbitcoins_test.go:106]: could add t.Parallel() here

**Review by MadCozBadd — APPROVED** [review]
tACK!

**Review by gloriousCode — APPROVED** [review]
tACK! 👍 💥 🌯 🥑 🧀

File: exchanges/localbitcoins/localbitcoins.go [hunk 2]
```diff
@@ -768,7 +764,13 @@ func (l *LocalBitcoins) SendAuthenticatedHTTPRequest(ep exchange.URL, method, pa
 	headers["Content-Type"] = "application/x-www-form-urlencoded"
 
 	if l.Verbose {
-		log.Debugf(log.ExchangeSys, "Sending POST request to `%s`, path: `%s`, params: `%s`.", endpoint, path, encoded)
+		log.Debugf(log.ExchangeSys, "%s Sending `%s` request to `%s`, path: `%s`, params: `%s`.",
```
Comment (shazbert) [inline exchanges/localbitcoins/localbitcoins.go:767]: [DEBUG]|EXCHANGE| 26/02/2021 10:02:26 |LocalBitcoins Sending `GET` request to `https://localbitcoins.com`, path: `/api/wallet-balance/`, params: ``.

👍

**Review by shazbert — APPROVED** [review]
tACK!

**Pull request merged**